skimage/util/arraycrop.py

```diff
@@ -4,24 +4,23 @@
 """
 
 import numpy as np
-from numpy.lib.arraypad import _as_pairs
 
 __all__ = ['crop']
 
 
-def crop(ar, crop_width, copy=False, order='K'):
-    """Crop array `ar` by `crop_width` along each dimension.
+def crop(array, crop_width, copy=False, order='K'):
```

Comment:
This is a proper renaming, but this will break the API. Reverting this change is safer.